app/routes/_extras.blog.rss[.xml].tsx

```diff
@@ -0,0 +1,46 @@
+import type { LoaderFunction } from "@remix-run/node";
+import { getBlogPostListings } from "~/lib/blog.server";
+
+const getCData = (text: string) => {
+  return `<![CDATA[${text}]]>`;
+};
+
+export const loader: LoaderFunction = async () => {
+  const blogUrl = `https://remix.run/blog`;
+  const posts = await getBlogPostListings();
+
+  const rss = `
+    <rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom">
+      <channel>
+        <title>Remix Blog</title>
+        <link>${blogUrl}</link>
+        <atom:link href="${blogUrl}/rss.xml" rel="self" type="application/rss+xml" />
+        <description>Thoughts about building excellent user experiences with Remix.</description>
+        <language>en-us</language>
+        <generator>https://github.com/remix-run/remix</generator>
+        <copyright>© Shopify, Inc.</copyright>
+        ${posts
+          .map(post =>
+            `
+            <item>
+              <title>${getCData(post.title)}</title>
+              <description>RFC-822 date: ${getCData(post.summary)}</description>
+              <pubDate>${new Date(post.dateDisplay).toUTCString()}</pubDate>
+              <link>${blogUrl}/${post.slug}</link>
+              <guid>${blogUrl}/${post.slug}</guid>
+            </item>
+          `.trim(),
+          )
+          .join('\n')}
+      </channel>
+    </rss>
+  `.trim();
+
+
+  return new Response(rss, {
+    headers: {
+      "Content-Type": "application/xml",
+      "Cache-Control": "public, max-age=0",
```

Comment:
Any reason to not add a max-age value here? I've never setup an RSS feed, so there very well may be a reason. Otherwise I would just do the following

```suggestion
      "Cache-Control": CACHE_CONTROL.DEFAULT
```

You'll need to import CACHE_CONTROL of course

Reply:
I'm truly sorry! I hadn't realized the existence of `CACHE_CONTROL.default` 🙇!
I fixed it now 👍

https://github.com/remix-run/remix-website/pull/184/commits/41b72c6290cc709f0f453467b4efb4d588343ef2

Reply:
Haha no need to apologize, you had no reason to know about this. Thanks for the update 🙂

I'll do a little more checking and then merge this at some point today 🎉